composer.json

```diff
@@ -10,20 +10,23 @@
         }
     ],
     "require": {
+        "ezsystems/ezpublish-kernel": "~5.1 || ^6.7",
         "guzzlehttp/guzzle": "~5.0|~6.0",
         "components/handlebars.js": "~3.0.0",
-        "symfony/symfony": "^2.7.7"
+        "symfony/symfony": "^2.7.7 || ^3.3"
```

Comment:
Backported to branch `1.2` of this package. Any objections @andrerom @SylvainGuittard @damianz5 ?

Reply:
Nope

Reply:
Nope. 
But why `3.3`? ezplatform repo has (on 2.0 branch) `3.4@beta`.

Reply:
Saw your comment on https://github.com/ezsystems/EzSystemsRecommendationBundle/pull/99#discussion_r149444813.
Makes totally sense :)